db/schema.rb

```diff
@@ -10,20 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170917153509) do
+ActiveRecord::Schema.define(version: 20170918125918) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "account_domain_blocks", id: :serial, force: :cascade do |t|
```

Comment:
The removal of id: serial seems incorrect here? Is this a consequence of changing to bigints or is it a schema change left over from the other PR?

Reply:
It's confusing, but no, this is actually "correct". Rails had written out `id: :serial` chunks for these tables to specify that a 32-bit `serial` column was to be used, rather than the 64-bit `bigserial` that is now the default. Given that these tables have now switched to 64-bit columns, it no longer specifies the type, as it will default to `bigserial`.

(And just to make sure, I didn't edit `db/schema.rb` by hand, this file was the result of a `rake db:drop && rake db:create && rake db:migrate`.)

Reply:
great, just wanted to make sure that there weren't migrations here on accident because the old one wasn't dropped.